test/TextToBinary.cpp

```diff
@@ -62,6 +62,27 @@ TEST(GetWord, Simple) {
   EXPECT_EQ("abc", AssemblyContext(AutoText("abc\n"), nullptr).getWord());
 }
 
+TEST(GetWord, MultipleWords) {
```

Comment:
Reasonable. Removed different whitespaces since they are tested in the previous test.